libmamba/src/core/solver.cpp

```diff
@@ -28,11 +28,11 @@
 namespace mamba
 {
     MSolver::MSolver(MPool pool, const std::vector<std::pair<int, int>> flags)
```

Comment:
- No need for `const` for a variable passed by value (and this actually can hits performance, see below).
- Not sure why this compiles since the constructor is declare as:
   ` MSolver(MPool, std::vector<std::pair<int, int>>)`